pkg/codegen/python/gen.go

```diff
@@ -2117,8 +2117,12 @@ func genPackageMetadata(
 
 	// Finally, the actual setup part.
 	fmt.Fprintf(w, "setup(name='%s',\n", pyPkgName)
-	if pythonRequires != "" {
-		fmt.Fprintf(w, "      python_requires='%s',\n", pythonRequires)
+	// Remind the user that the schema should specify the minimum
```

Comment:
I'm not really sure it should actually. We control code gen, we should know what the minimum python version is, why does the user need to set anything. This field should probably just be deprecated.

Reply:
I have our largest customer in mind. We've said we want to only support Python versions until they sunset. But some customers with internal providers might have different requirements.

Reply:
Leave it in as an override then, but it should default to something sensible and shouldn't error at every standard user.

Reply:
Ok, when we generate `pyproject.toml`, I'll add a default value!

Reply:
I added a `defaultMinimumVersion` constant to use as the default. The user generating the package should get an `Info`-level message that they failed to set the minimum package version. 👍🏻